src/Apps/Order2/Routes/Checkout/Components/OfferStep/Forms/Order2ExactPriceOfferForm.tsx

```diff
@@ -0,0 +1,270 @@
+  Flex,
+  RadioGroup,
+  Spacer,
+  Text,
+  Button,
+  TextArea,
+} from "@artsy/palette"
+import { OfferInput } from "Apps/Order/Components/OfferInput"
+import { appendCurrencySymbol } from "Apps/Order/Utils/currencyUtils"
+import type { OfferFormComponentProps } from "Apps/Order2/Routes/Checkout/Components/OfferStep/types"
+import type { Order2ExactPriceOfferForm_order$key } from "__generated__/Order2ExactPriceOfferForm_order.graphql"
+import { useEffect, useState } from "react"
+import { graphql, useFragment } from "react-relay"
+
+interface PriceOption {
+  key: string
+  value: number
+  description: string
+}
+
+interface Order2ExactPriceOfferFormProps extends OfferFormComponentProps {
+  order: Order2ExactPriceOfferForm_order$key
+}
+
+export const Order2ExactPriceOfferForm: React.FC<
+  Order2ExactPriceOfferFormProps
+> = ({
+  order,
+  offerValue,
+  offerNoteValue,
+  formIsDirty,
+  isSubmittingOffer,
+  onOfferValueChange,
+  onOfferNoteChange,
+  onOfferInputFocus,
+  onContinueButtonPressed,
+}) => {
+  const orderData = useFragment(FRAGMENT, order)
+  const artwork = orderData.lineItems?.[0]?.artwork
+  const listPriceMajor =
+    artwork?.listPrice?.__typename === "Money"
+      ? artwork.listPrice.major
+      : artwork?.listPrice?.__typename === "PriceRange"
+        ? artwork.listPrice.maxPrice?.major
+        : null
+
+  const priceOptions: PriceOption[] = listPriceMajor
+    ? [
+        {
+          key: "price-option-max",
+          value: Math.round(listPriceMajor),
+          description: "List price",
+        },
+        {
+          key: "price-option-mid",
+          value: Math.round(listPriceMajor * 0.9), // 10% below
+          description: "10% below list price",
+        },
+        {
+          key: "price-option-min",
+          value: Math.round(listPriceMajor * 0.8), // 20% below
+          description: "20% below list price",
+        },
+      ]
+    : []
+
+  const getInitialState = () => {
+    if (!offerValue)
+      return {
+        selectedRadio: undefined,
+        customValue: undefined,
+      }
+
+    const matchingOption = priceOptions.find(
+      option => option.value === offerValue,
+    )
+    if (matchingOption) {
+      return {
+        selectedRadio: matchingOption.key,
+        customValue: undefined,
+      }
+    } else {
+      return {
+        selectedRadio: "price-option-custom",
+        customValue: offerValue,
+      }
+    }
+  }
+
+  const initialState = getInitialState()
+  const [selectedRadio, setSelectedRadio] = useState<string | undefined>(
+    initialState.selectedRadio,
+  )
+  const [customValue, setCustomValue] = useState<number | undefined>(
+    initialState.customValue,
+  )
+
+  // Update parent component when custom values change
+  useEffect(() => {
+    if (selectedRadio === "price-option-custom") {
+      onOfferValueChange(customValue || 0)
+    }
+  }, [customValue, selectedRadio, onOfferValueChange])
+
+  const formatCurrency = (amount: number) => {
```

Comment:
We can probably move this `formatCurrency` to this shared `currencyUtils ` - seems to be repeated in several places.